parliament/statement.py

```diff
@@ -955,4 +955,4 @@ def analyze_statement(self):
                 "RESOURCE_STAR", detail=sorted(self.resource_star), location=self.stmt
             )
 
-        return True
+        return not has_malformed_resource
```

Comment:
I think this line is the only change the PR mentioned?

Reply:
Yup, I added a couple of lines about the 2 other commits now, let me know if you'd prefer I mention it in the PR title.